test/unit/org/apache/cassandra/transport/SerDeserTest.java

```diff
@@ -96,7 +94,7 @@ public void collectionSerDeserTest() throws Exception
         for (Integer i : l)
             lb.add(Int32Type.instance.decompose(i));
 
-        assertEquals(l, lt.getSerializer().deserialize(CollectionSerializer.pack(lb, lb.size())));
+        assertEquals(l, lt.compose(lt.pack(lb)));
```

Comment:
Since we are here, we can remove the unneeded `throws Exception` at the method signature.